packages/mui-base/src/SliderUnstyled/SliderUnstyled.types.ts

```diff
@@ -212,9 +226,71 @@ export type ExtendSliderUnstyled<M extends OverridableTypeMap> = OverridableComp
   ExtendSliderUnstyledTypeMap<M>
 >;
 
-type SliderUnstyledProps<
+export type SliderUnstyledProps<
   D extends React.ElementType = SliderUnstyledTypeMap['defaultComponent'],
   P = {},
 > = OverrideProps<SliderUnstyledTypeMap<P, D>, D>;
 
-export default SliderUnstyledProps;
+export type SliderUnstyledRootSlotProps = UseSliderRootSlotProps & {
+  children: React.ReactNode;
+  className: string;
+  ownerState: SliderUnstyledOwnerState;
+};
+
+export type SliderUnstyledTrackSlotProps = UseSliderRootSlotProps & {
```

Comment:
The `UseSliderRootSlotProps` also contains the `ref` no? Would that be a problem? Also, why would we have here this and not in the rest of the slots, for example `SliderUnstyledRailSlotProps`?

Reply:
Too much copy-pasta. SliderUnstyledTrackSlotProps does not extend UseSliderRootSlotProps. The SliderUnstyled implementation is still JS, so TS did not catch the error.